### PR TITLE
Adding comments to Update main.go

### PR DESCRIPTION
**Pull request by Gordiancyber**
adding comments to provide explanations for each line of code in the Go script to make it easier to understand the purpose of each part of the code.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,131 +1,136 @@
...
+			_, message, err := c.ReadMessage()
+			if err != nil {
+				log.Println("read:", err)
+				return
+			}
+			messageOut <- string(message)
+		}
+	}()
+	x := <-messageOut
+	fmt.Println(x)
```
Comment (stevenanthonyrevo) [inline main.go:44]: Can you remove the printed variable?

**Review by stevenanthonyrevo — CHANGES_REQUESTED** [review]
@Gordiancyber Looks good! 

Close to merging to master however can the `Print` log I mentioned before removed?

File: main.go (hunk 1 above)
Comment (Gordiancyber) [reply main.go:44]: yes, I would do that just now

Comment (Gordiancyber) [reply main.go]: > Can you remove the printed variable?

The printed variable has been removed. Let me know if you need any additional changes.

**Review by stevenanthonyrevo — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (stevenanthonyrevo) [reply main.go]: @Gordiancyber, Great work! Have a Happy Hacktoberfest!

**Pull request merged**